tests/common/iterators_support.hpp

```diff
@@ -113,7 +113,7 @@ class input_it {
 	using pointer = typename std::iterator_traits<It>::pointer;
 	using reference = typename std::iterator_traits<It>::reference;
 
-	input_it() = delete;
+	input_it() : _it(){};
```

Comment:
Why you changed this? If this iterator should have non-deleted contructor what about output_it?

Reply:
Default construction of input_it is required by tests included in string_assing/iterator.pass.cpp (IIter() - lines 382, 386, 389). Libcxx implementation of input iterator defines default ctor, but ours not (with no reason as far as I know). Apart of that I will define def. ctor. for output_it as well.